firebase_admin/_messaging_utils.py

```diff
@@ -98,7 +98,8 @@ class AndroidNotification:
             the user clicks it (optional).
         event_timestamp: For notifications that inform users about events with an absolute time
             reference, sets the time that the event in the notification occurred as a
-            ``datetime.datetime`` instance. Notifications in the panel are sorted by this time
+            ``datetime.datetime`` instance. If the ``datetime.datetime`` instance is naive it will
+            be assumed to be in the UTC timezone. Notifications in the panel are sorted by this time
```

Comment:
FYI @egilmorez this is the API docs change. Thanks!

Reply:
Not critical, butI wonder if there is a more terse way to say "it will be assumed to be in the" . . . would "naive, it defaults to" be correct?  Maybe "naive, it is set to" ?

If those aren't accurate, this is fine as/is.